fv3gfs-util/fv3gfs/util/partitioner.py

```diff
@@ -568,22 +592,21 @@ def global_extent(self, rank_metadata: QuantityMetadata) -> Tuple[int, ...]:
             rank_metadata.dims, rank_metadata.extent, self.layout
         )
 
-    def subtile_extent(self, cube_metadata: QuantityMetadata) -> Tuple[int, ...]:
+    def subtile_extent(self, cube_metadata: QuantityMetadata, rank: int, overlap: bool) -> Tuple[int, ...]:
         """Return the shape of a single rank representation for the given dimensions."""
         if cube_metadata.dims[0] != constants.TILE_DIM:
             raise NotImplementedError(
                 "currently only supports tile dimension {constants.TILE_DIM} as the "
                 "first dimension, got dims {cube_metadata.dims}"
             )
-        return rank_extent_from_tile_metadata(
-            cube_metadata.dims[1:], cube_metadata.extent[1:], self.layout
-        )
+
+        return self.tile.subtile_extent(cube_metadata, rank, overlap, cube_metadata_offset=1)
 
     def subtile_slice(
         self,
-        rank: int,
         global_dims: Sequence[str],
         global_extent: Sequence[int],
+        rank: int,
```

Comment:
Please keep call signatures the same where possible.

Reply:
Reverted the argument order change.